spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'with Authenticated User'
+
+  describe 'GET #index' do
+    context 'when fetching all the books' do
+      let(:books) { create_list(:book, 3) }
+
+      before do
+        get :index
```

Comment:
If you think about it, this is your subject, so instead  of putting it in a before you can make a subjet like this:

```
subject(:http_request) { get :index }
```

And change the response in the examples

Reply:
DONE